portfolio/src/main/webapp/script.js

```diff
@@ -53,20 +53,23 @@ function openContact(){
   window.open("contact_me.html")
 }
 
-async function getWordsFromServer(){
+async function getMessageFromServer(){
+  /** 
   const serverResponse = await fetch("/data");
   console.log(serverResponse);
   const responseToJson = await serverResponse.json();
   
-  //create object with response from server
-  wordsArray = responseToJson;
 
-  //access different elements using [] since it's an array
-  console.log(wordsArray[0]);
-  console.log(wordsArray[1]);
-  console.log(wordsArray[2]);
-
-  //Add to page 
+  Add to page
   const message = document.getElementById("greeting-container");
-  message.innerText += wordsArray.join(' ');
+  message.innerText = responseToJson;
+  */
+}
+function validateForm() {
```

Comment:
Great to put a function to validate the input from users (or external sources). Keep the good habit:)